### PR TITLE
Upgrade dependencies and update README

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,156 @@
...
+
+```html
+<h1 class="o-editorial-typography-heading-level-1">Heading level 1</h1>
+<h2 class="o-editorial-typography-heading-level-2">Heading level 2</h2>
+<h3 class="o-editorial-typography-heading-level-3">Heading level 3</h3>
+<h4 class="o-editorial-typography-heading-level-4">Heading level 4</h4>
+<h5 class="o-editorial-typography-heading-level-5">Heading level 5</h5>
+```
+
+Heading level 1 is a suitable for most cases but for more emphasis use `o-editorial-typography-headline` instead.
```
Comment (notlee) [inline README.md]:
```suggestion
Heading level 1 is suitable for most cases but for more emphasis use `o-editorial-typography-headline` instead.
```

**Review by rowanmanning — APPROVED** [review]
🎉

File: README.md [hunk 2]
```diff
@@ -1,34 +1,156 @@
...
+#### Headings
+
+Headings 1-6 are provided with the class `o-editorial-typography-heading-level-[level]`:
+
+```html
+<h1 class="o-editorial-typography-heading-level-1">Heading level 1</h1>
+<h2 class="o-editorial-typography-heading-level-2">Heading level 2</h2>
+<h3 class="o-editorial-typography-heading-level-3">Heading level 3</h3>
+<h4 class="o-editorial-typography-heading-level-4">Heading level 4</h4>
+<h5 class="o-editorial-typography-heading-level-5">Heading level 5</h5>
```
Comment (JakeChampion) [inline README.md:24]: I think we should to finish off the list with the h6 here, as it is the only one missing in the example
```suggestion
<h5 class="o-editorial-typography-heading-level-5">Heading level 5</h5>
<h6 class="o-editorial-typography-heading-level-6">Heading level 6</h6>
```

File: src/scss/_mixins.scss [hunk 3]
```diff
@@ -5,7 +5,7 @@
 ///     }
 @mixin oEditorialTypographyBody() {
 	@include oTypographySerif($scale: (default: 1, XL: 2), $line-height: 1.6);
-	color: oColorsGetColorFor('body', 'text');
```
Comment (JakeChampion) [inline src/scss/_mixins.scss:8]: How did this work?! it never directly depended on o-colors! 😱

Comment (notlee) [reply src/scss/_mixins.scss:8]: By the magic of _indirectly_ depending on o-colors!  Let's get ourselves and the ft at large to use the new module system

**Pull request merged**